src/game/raids/index.ts

```diff
@@ -38,6 +38,15 @@ export interface Phase extends PhaseConfig {
   start: number[];
   end: number[];
 }
+export interface Dungeon {
+  id: number;
+  name: string;
+  background?: string;
+  backgroundPosition?: string;
+  headshot?: string;
+  icon?: string;
+  fight: unknown;
```

Comment:
I saw that the Raid interface declares this as an EncounterConfig but since there is only an empty set of { } I figured I'd put this as unknown right now.